### PR TITLE
unity-setup@v1.0.6

### PR DESCRIPTION
**Pull request by StephenHodgson**
- improved `version-file` search, and added `None` option when creating new projects or setting `UNITY_PROJECT_PATH` later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ steps:
 
 | name | description | required |
 | ----------- | ----------- | ----------- |
-| `version-file` | Specify a path to search for the unity project version text file. Useful if there are multiple projects in a single repo. | false |
+| `version-file` | Specify a path to search for the unity project version text file. Useful if there are multiple projects in a single repo. Pass `None` if creating a new project to skip file search. | false |
```
Comment (afarchy) [inline README.md:54]: Based on this description I don't understand when I would use 'None'

Comment (afarchy) [inline README.md:54]: how about changing this to just project-path, and deriving the version file path from there?

**Review by afarchy — APPROVED** [review]
looks fine but see comments

File: README.md (hunk 1 above)
Comment (StephenHodgson) [reply README.md:54]: When creating a new project or skipping the file search option.

Comment (StephenHodgson) [reply README.md:54]: that'd be a breaking change imho, and would rather keep it like it is to onboard people using the older versions of this action.

Comment (StephenHodgson) [reply README.md:54]: I don't really think it matters too much, if you have the project path, you can get the version file, and if you have the version file you can get the project path.

**Pull request merged**